Dockerfile.cypress

```diff
@@ -1,3 +1,20 @@
+FROM golang:1.15.8 AS base
+
+WORKDIR /easi/
+
+FROM base AS modules
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
```

Comment:
I don't know much about docker but! Is this going to be prohibitively slow?

Reply:
It will be a little bit slower, but I don't think it's enough to be an issue. It's going to be difficult to optimize because we need a Go executable in the Cypress container now. But luckily the Go compiler is pretty fast.